AvalonStudio/AvalonStudio.Extensibility/IExtensionPackage.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+
+namespace AvalonStudio.Extensibility
+{
+    public interface IExtensionPackage
+    {
+        string Name { get; }
+        string Version { get; }
```

Comment:
Do we need a description?

Reply:
I tried to define a simple model, but forgot to add a property for `Description`, I'll do that. Also, I'm thinking if I should rename `IExtensionPackage` to `IExtensionManifest` and `ExtensionPackage` to `ExtensionManifest`, what do you think?

Reply:
Yes I think so, a package would be the thing all this gets downloaded in, like a nuget package, these I suppose are manifests that explain the content and how it can be used